### DISCUSSION TITLE
hear ye hear ye: dogs >>>>> cats

### DISCUSSION THREAD

**Discussion by jamieshark** [jamieshark/goothoob#6] [Announcements]
unless otherwise specified nothing wrong with cats just statin fax

**Comment by robertnopickle** [comment]
cats are more civilized.  i mean, who goes to the bathroom outside?

**Reply by jamieshark** [reply]
cats literally go to the bathroom in sand which is found outside so what's your point

**Reply by robertnopickle** [reply]
apart from being needy, name one thing a dog does better than a cat?
a cat trained to attack, a guard cat if-you-will, is 1000 times more scary than any guard dog.  a whirlwind of razorblades.

**Reply by jamieshark** [reply]
granted that perhaps a gale of butter knives is somewhat threatening, I would also like to point out that with their 9 lives, cats may never die, and do you or do you not want a ride-or-die for a best friend (i.e. dogs)
